prismic-model/json/pages.json

```diff
@@ -364,6 +364,35 @@
               }
             }
           },
+          "tagList": {
+            "type": "Slice",
+            "fieldset": "Tag List",
+            "non-repeat": {
+              "title": {
+                "type": "StructuredText",
+                "config": {
+                  "label": "Title",
+                  "single": "heading2"
+                }
+              }
+            },
+            "repeat": {
+              "link": {
+                "type": "Link",
+                "config": {
+                  "label": "Link",
+                  "select": "web",
+                  "customtypes": []
+                }
+              },
+              "linkText": {
+                "type": "Text",
+                "config": {
+                  "label": "Link text"
+                }
+              }
+            }
+          },
```

Comment:
We need to run through and get everything up to date, these things foggy the PRs.